src/docs/deployment/linux.md

```diff
@@ -262,6 +300,36 @@ parts:
     flutter-target: lib/main.dart # The main entry-point file of the application
 ```
 
+
+## Desktop file and icon
+: Desktop entry files are used to add an application 
```

Comment:
After a header, the following text should be a paragraph, not a definition list.

Reply:
Thank you, changes commited